CreeDictionary/API/apps.py

```diff
@@ -56,47 +65,45 @@ def read_morpheme_rankings():
             Wordform.MORPHEME_RANKINGS[morpheme] = float(freq)
 
 
-def initialize_affix_search():
+def initialize_affix_search() -> None:
     """
-    build tries and attach to Wordform class to facilitate prefix/suffix search
+    Build tries and attach to Wordform class to facilitate prefix/suffix search
     """
     logger.info("Building tries for affix search...")
-    from .models import Wordform
-    from .models import EnglishKeyword
+    from .models import (
+        EnglishKeyword,
+        Wordform,
+        set_affix_searcher_for_cree,
+        set_affix_searcher_for_english,
+    )
 
     try:
-        lowered_no_diacritics_cree_with_id = [
-            (remove_cree_diacritics(text.lower()), wf_id)
-            for text, wf_id in Wordform.objects.filter(is_lemma=True).values_list(
-                "text", "id"
-            )
-        ]
-
-        lowered_english_keywords_with_wf_id = [
-            (kw, wf_id)
-            for kw, wf_id in EnglishKeyword.objects.all().values_list(
-                "text", "lemma__id"
-            )
-        ]
-
+        Wordform.objects.count()
+    except OperationalError:
         # apps.py will also get called during migration, it's possible that neither Wordform table nor text field
         # exists. Then an OperationalError will occur.
-    except OperationalError:
+        logger.exception("Cannot build tries: Wordform table does not exist (yet)!")
         return
 
-    Wordform.affix_searcher = AffixSearcher(
-        lowered_no_diacritics_cree_with_id + lowered_english_keywords_with_wf_id
-    )
+    set_affix_searcher_for_cree(AffixSearcher(fetch_cree_lemmas_with_ids()))
+    set_affix_searcher_for_english(AffixSearcher(fetch_english_keywords_with_ids()))
```

Comment:
This is super nit-picky:
```suggestion
    set_affix_searcher_for_english(AffixSearcher(fetch_english_lemmas_with_ids()))
```
The use of "lemmas" in one function and "keywords" in the other makes me think they do different things. If that's the case, ignore this suggestion.

Reply:
They do different things! So this confusing wording is entirely intentionally. The results that can be matched are _lemmas_ — base form of words defined in the dictionary and FSTs, while the results that can be matched for English are _keywords_, mined from the dictionary definitions.